snmp/datadog_checks/snmp/data/default_profiles/_hp-compaq-health.yaml

```
@@ -181,14 +182,42 @@ metrics:
       OID: 1.3.6.1.4.1.232.6.2.9.3
       name: cpqHeFltTolPowerSupplyTable
     symbols:
-      - OID: 1.3.6.1.4.1.232.6.2.9.3.1.5
-        name: cpqHeFltTolPowerSupplyStatus
+      - name: cpqHeFltTolPowerSupply
+        constant_value_one: true
```

```suggestion
      - OID: 1.3.6.1.4.1.232.6.2.9.3.1.5
        name: cpqHeFltTolPowerSupplyStatus
      - name: cpqHeFltTolPowerSupply
        constant_value_one: true
```

We should probably keep the existing `cpqHeFltTolPowerSupplyStatus` since removing it can be a breaking change for users using it.